source/Ship.h

```diff
@@ -310,6 +310,8 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	// Get the cost of making a jump of the given type (if possible).
 	double HyperdriveFuel() const;
 	double JumpDriveFuel(double jumpDistance = 0.) const;
+	enum JumpType{ HYPERDRIVE, JUMPDRIVE, NONE };
```

Comment:
Can you:

- Make this an `enum class`
- Move it up with the other class definitions
- Not caps the values

thx

Reply:
done